templates/haproxy.cfg.j2

```diff
@@ -12,40 +12,68 @@ global
   daemon
 
 defaults
-  log global
-  mode  http
-  option  httplog
-  option  dontlognull
+    log global
+    mode  http
+{% if haproxy_config is defined and haproxy_config.defaults is defined and haproxy_config.defaults.options is defined %}
+{% for value in haproxy_config.defaults.options %}
+    option {{ value }}
+{% endfor %}
+{% endif %}
+    option  httplog
+    option  dontlognull
 {% if haproxy_version == '1.4' %}
-        contimeout 5000
-        clitimeout 50000
-        srvtimeout 50000
+    contimeout 5000
+    clitimeout 50000
+    srvtimeout 50000
 {% else %}
-        timeout connect 5000
-        timeout client 50000
-        timeout server 50000
+    timeout connect 5000
+    timeout client 50000
+    timeout server 50000
 {% endif %}
 {% if ansible_os_family == 'Debian' %}
-  errorfile 400 /etc/haproxy/errors/400.http
```

Comment:
Looks like @Grobix added extra spaces in the YAML; this needs to be fixed so the PR doesn't make whitespace inconsistent.